### ISSUE TITLE
add a redis backend

### ISSUE THREAD

**Issue by fruch** [shaypal5/cachier#4]
redis is the king of cache, a backend for it can be cool
(on my todo list, down on the bottom of it)

**Comment by shaypal5** [comment]
Hmm, that's right!

Well, hopefully it would be easy with core a architecture.

**Comment by a904guy** [comment]
Memcached as well, working on an adapter core now.

**Comment by shaypal5** [comment]
@a904guy Sounds great! Can you please open a new issue with the title "Add support for Memcached cores" or something similar, so we can track these separately?

**Comment by advance512** [comment]
Any update on this?

**Comment by shaypal5** [comment]
@advance512  I never heard from anyone about this since. If you know of a good (or even official) Python driver and an easy and free way to fabricate a light Redis instance I could look into it myself. Maybe.

**Comment by fruch** [comment]
@shaypal5 https://pypi.python.org/pypi/fakeredis, and you can use the formal docker image.
(I you don't have a docker setup, I highly recommend giving it a try, if you need help with that give me a ping)

**Comment by dmwyatt** [comment]
Did anyone ever make something like this?

**Comment by shaypal5** [comment]
Nope. Again. Shouldn't be hard based on the MongoDB core, but someone with some time and interest will have to contribute this.
